suggestedimages/external/__init__.py

```diff
@@ -0,0 +1,11 @@
+import requests_cache
+requests_cache.install_cache()
+
+from .yso import YSO
+
+
+by_language = {
+    'fi': [
+        YSO('fi')
+    ]
+}
```

Comment:
What by language?

`something_by_language` could offer more context

Reply:
Yes, you're right. I renamed it and the whole module.